tests/base.py

```diff
@@ -21,7 +31,7 @@ class GA4Base(BaseCase):
 
     HASHED_KEYS = "default-hashed-keys"
     # REPLICATION_KEY_FORMAT = "%Y-%m-%dT00:00:00.000000Z"
-    CONVERSION_WINDOW = 10
+    CONVERSION_WINDOW = 30
```

Comment:
Remove commented code if not kept deliberately.

Reply:
Beat you to it :)